griptape/utils/chat.py

```diff
@@ -25,9 +27,16 @@ def start(self) -> None:
 
             if question.lower() in self.exit_keywords:
                 self.output_fn(self.exiting_text)
-
                 break
             else:
                 self.output_fn(self.processing_text)
 
-            self.output_fn(f"{self.response_prefix}{self.structure.run(question).output_task.output.to_text()}")
+            if self.structure.stream:
```

Comment:
Let's update this to `self.structure.prompt_driver.stream` so that it works with all Prompt Drivers.